hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/MockPipelineManager.java

```diff
@@ -296,6 +297,11 @@ public Map<String, Integer> getPipelineInfo() {
     return null;
   }
 
+  @Override
+  public Map<String, String[]> getPipelineLeaders() throws NotLeaderException {
```

Comment:
`throws NotLeaderException` is redundant

Reply:
Thanks for the find, I have removed it !!